### PR TITLE
Add exact match on topics

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds an exact match on topics for ZTF, PGIR, and WINTER.

### CODE REVIEW

File: kowalski/alert_broker_pgir.py [hunk 1]
```diff
@@ -526,15 +527,17 @@ def watchdog(obs_date: str = None, test: bool = False):
...
             if not test:
-                # Production Kafka stream at IPAC
-                kafka_cmd = [
-                    os.path.join(config["path"]["kafka"], "bin", "kafka-topics.sh"),
-                    "--zookeeper",
-                    config["kafka"]["zookeeper"],
-                    "-list",
-                ]
+                # as of 20210722, the naming convention is pgir_%Y%m%d_programidN
+                topics_tonight = [f"pgir_{datestr}"]
```
Comment (guynir42) [inline kowalski/alert_broker_pgir.py:539]: This doesn't need a program id?

**Review by guynir42 — APPROVED** [review]
Two small questions, just to make sure.

File: kowalski/alert_broker_pgir.py (hunk 1 above)
Comment (mcoughlin) [reply kowalski/alert_broker_pgir.py:539]: Yeah no program IDs for non-ZTF.

File: kowalski/alert_broker_ztf.py [hunk 2]
```diff
@@ -488,25 +495,20 @@ def watchdog(obs_date: str = None, test: bool = False):
...
-                and ("pgir" not in t)
-            ]
+                topics_tonight = [
+                    t
+                    for t in topics
+                    if (datestr in t)
+                    and ("programid" in t)
+                    and ("zuds" not in t)
+                    and ("pgir" not in t)
+                ]
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply kowalski/alert_broker_ztf.py:511]: Yeah because of how the testing is setup.

**Review by virajkaram — APPROVED** [review]

**Pull request merged**